airbyte-integrations/connectors/source-notion/integration_tests/abnormal_state.json

```diff
@@ -31,5 +31,57 @@
         "name": "blocks"
       }
     }
+  },
+  {
+    "type": "STREAM",
+    "stream": {
+      "stream_descriptor": { "name": "comments", "namespace": null },
+      "stream_state": {
+        "states": [
+          {
+            "partition": {
+              "id": "9cca331a-1edb-4120-9a86-cf40214fac37",
+              "parent_slice": {}
```

Comment:
Should we cover a case where `parent_slice` isn't empty?

Reply:
The empty parent slices accurately reflect the expected state values; the partitioning is based only on a parent_id (ie each partition consists of all comments found within a specific page, with no additional layers of parent slicing). I don't think there would be any harm logic-wise in mocking a parent_slice value here, but I'd be more concerned about causing any confusion for a future dev